modules/sdk/project-templates/project-templates/src/main/resources/com/liferay/project/templates/jakarta-dependencies/jakarta-dependencies.properties

```diff
@@ -0,0 +1,7 @@
+javax.annotation_javax.annotation-api=group: "jakarta.annotation", name: "jakarta.annotation-api", version: "2.1.1"
+javax.portlet_portlet-api=group: "com.liferay.jakarta.portlet", name: "com.liferay.jakarta.portlet-api", version: "4.0.0"
+javax.servlet_javax.servlet-api=group: "jakarta.servlet", name: "jakarta.servlet-api", version: "6.0.0"
+javax.validation_validation-api=group: "jakarta.validation", name: "jakarta.validation-api", version: "3.1.0"
+javax.ws.rs_javax.ws.rs-api=group: "com.liferay", name: "jakarta.ws.rs", version: "3.1.0.LIFERAY-PATCHED-1"
+javax.xml.bind_jaxb-api=group: "jakarta.xml.bind", name: "jakarta.xml.bind-api", version: "4.0.2"
+org.osgi_org.osgi.service.jaxrs=group: "com.liferay", name: "org.osgi.service.jaxrs", version: "1.0.0.JAKARTA-LIFERAY-PATCHED-1"
```

Comment:
Can we switch to the colon-separated format instead?

```
javax.annotation:javax.annotation-api=jakarta.annotation:jakarta.annotation-api:2.1.1
```

Reply:
This matches other similar properties files in portal and is more likely to be accepted by Brian.

Reply:
Updated the values to be colon-separated format and adapted my code to match. The key, however, I still split with an underscore. Properties read the first ":" or "=" to identify the start of a value, so using ":" in the key would break things